core-java-modules/core-java-collections-6/src/test/java/com/baeldung/checkedcollections/DataProcessorUnitTest.java

```diff
@@ -0,0 +1,33 @@
+package com.baeldung.checkedcollections;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DataProcessorUnitTest {
```

Comment:
There's no need to make the class public.

Reply:
> Done

Reply:
I've realized you have removed the modifier for `DataProcessor` but not for `DataProcessorUnitTest`

Reply:
Done the changes to the test.